src/infer/inferOs.js

```diff
@@ -11,7 +11,7 @@ function inferPlatform() {
 
 function inferArch() {
   const arch = os.arch();
-  if (arch !== 'ia32' && arch !== 'x64') {
+  if (arch !== 'ia32' && arch !== 'x64' && arch !== 'arm') {
```

Comment:
@mdh34 👍, great then. Merging, thanks again for the contribution! Out of curiosity, on what kind of device do you use Nativefier? Raspberry pi?